pkg/clusteragent/autoscaling/workload/model/pod_autoscaler_test_utils.go

```diff
@@ -31,6 +31,7 @@ type FakePodAutoscalerInternal struct {
 	SettingsTimestamp              time.Time
 	CreationTimestamp              time.Time
 	ScalingValues                  ScalingValues
+	MainScalingValues              ScalingValues
```

Comment:
Don't you need fallback as well?

Reply:
Yes though the current test cases don't cover local fallback activation as `handleScaling` isn't tested - can add unit tests for the logic